<?xml version="1.0" encoding="UTF-8"?>
<class xmlns="http://xml.phpdox.net/src" full="Symfony\Component\Debug\Exception\ClassNotFoundException" namespace="Symfony\Component\Debug\Exception" name="ClassNotFoundException" abstract="false" final="false" start="19" end="33">
  <file path="/home/robert/Desktop/Hogeschool/2ejaars Development/DevBOpdrachten/project56/http/laravel/vendor/symfony/debug/Exception" file="ClassNotFoundException.php" realpath="/home/robert/Desktop/Hogeschool/2ejaars Development/DevBOpdrachten/project56/http/laravel/vendor/symfony/debug/Exception/ClassNotFoundException.php" size="828" time="2015-12-17T18:44:10+00:00" unixtime="1450377850" sha1="eac60d93bb51971870559a3d755bc9c754cb9b72" relative="vendor/symfony/debug/Exception/ClassNotFoundException.php"/>
  <docblock>
    <description compact="Class (or Trait or Interface) Not Found Exception."/>
    <author value="Konstanton Myakshin &lt;koc-dp@yandex.ru&gt;"/>
  </docblock>
  <extends full="Symfony\Component\Debug\Exception\FatalErrorException" namespace="Symfony\Component\Debug\Exception" name="FatalErrorException"/>
  <constructor name="__construct" start="21" end="32" abstract="false" final="false" static="false" visibility="public">
    <parameter name="message" byreference="false" type="{unknown}"/>
    <parameter name="previous" byreference="false" type="object">
      <type full="ErrorException" namespace="" name="ErrorException"/>
    </parameter>
  </constructor>
</class>
